readme.md

```diff
@@ -164,6 +164,10 @@ Comprehensive tools for Data Visualization.
   <sup>[[dep: matplotlib](#matplotlib)]</sup>
 - [mathematical](https://github.com/gjtorikian/mathematical) &mdash;
   PNG and MathML renderings for your equations.
+- [daru-view](https://github.com/sciruby/daru-view) &mdash;
+  daru-view is **one of the best** gem for easy and interactive plotting in web application
```

Comment:
This statement is very subjective :) And it doesn't give you any additional insights. Let the readers decide themselves what is good and what is not good for them.

Reply:
Thanks for review, I have changed it.